modules/misc/news.nix

```diff
@@ -1245,6 +1245,16 @@ in
           A new module is available: 'services.spotifyd'.
         '';
       }
+
+      {
```

Comment:
Generally speaking we don't add news entries just for new options so I would suggest removing it. Alternatively, add a condition to restrict it configurations where `services.emacs` is enabled…

Reply:
I'll remove it.